
syntax = "proto3";

message AddForwardRequest {
    string address = 1;
    int32 port = 2;
    string token = 3;
}

message RemoveForwardRequest {
    string address = 1;
    int32 port = 2;
    string token = 3;
}

message AddForwardResponse {
    string message = 1;
}

message RemoveForwardResponse {
    string message = 1;
}

service AlterForward {
    rpc AddForward(AddForwardRequest) returns (AddForwardResponse) {}
    rpc RemoveForward(RemoveForwardRequest) returns (RemoveForwardResponse) {}
}